backend/src/govsg/services/govsg.service.ts

```diff
@@ -140,8 +147,7 @@ const getLanguageCode = (language: string | undefined) => {
   return WhatsAppLanguages[key as keyof typeof WhatsAppLanguages]
 }
 
-const isUsingPrecallTemplate = async (govsgMessage: GovsgMessage) => {
-  const campaignId = govsgMessage.campaignId
+const isUsingPrecallTemplate = async (campaignId: number) => {
```

Comment:
Is the name of this function outdated?

Reply:
iirc I renamed this function in some other branch. I'll have to resolve the conflict later.